markhor_slam/README.md

```diff
@@ -0,0 +1,9 @@
+# markhor_slam
```

Comment:
Maybe a little line on how to use it or point it to documentation that tell you how